modules/json/json-parser.c

```diff
@@ -215,10 +215,21 @@ json_parser_process(LogParser *s, LogMessage **pmsg, const LogPathOptions *path_
   struct json_object *jso;
   struct json_tokener *tok;
 
+  msg_debug("  json-parser started message processing",
+            evt_tag_str ("input", input),
+            evt_tag_str ("prefix", self->prefix),
+            evt_tag_str ("marker", self->marker),
+            evt_tag_printf("msg", "%p", *pmsg));
   if (self->marker)
     {
       if (strncmp(input, self->marker, self->marker_len) != 0)
-        return FALSE;
+        {
+          msg_debug("  json-parser failed",
+                    evt_tag_str ("error", "marker not found"),
+                    evt_tag_str ("input", input),
```

Comment:
on the error field:"json marker not found at the beginning of the message"

Reply:
Thanks, fixed.